localstripe/resources.py

```diff
@@ -244,6 +244,67 @@ def do_expand(path, obj):
         return obj
 
 
+class Balance(object):
+    object = 'balance'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        self.object = self.object
+        self.livemode = False
+        self.type = 'balance'
```

Comment:
I do not see any `type` on the Balance object: https://stripe.com/docs/api/balance/balance_object